hack/aks/Makefile

```diff
@@ -96,6 +96,8 @@ overlay-byocni-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster
 		--pod-cidr 192.168.0.0/16 \
 		--vnet-subnet-id /subscriptions/$(SUB)/resourceGroups/$(GROUP)/providers/Microsoft.Network/virtualNetworks/$(VNET)/subnets/nodenet \
 		--no-ssh-key \
+		--kubernetes-version $(K8S_VER) \
```

Comment:
Just curious, For which sku do we need to specify k8s version ? I remember Ubuntu 18 had that requirement.

Reply:
Also updated, as it is no longer necessary for this change.